src/component/landing/index.jsx

```diff
@@ -30,6 +30,14 @@ const LandingPageView = () => {
       changeUberPrices(pricingToDisplay.message.uber);
       changeLyftPrices(pricingToDisplay.message.lyft);
     }
+    else {
+      changeUberPrices([]);
+      changeLyftPrices([]);
+      if (pricingToDisplay.status >= 400)
+      {
+        alert("Error: Bad Request");  
+      }
+    }
```

Comment:
I'm not entirely sure how I personally feel about an `alert()` message popping up on a bad request. I like the idea of showing an error message when the request does not succeed. If we are just going with a quick fix for the product demo, then sure, but if we can implement a mechanism that shows the status of the request, that'd be dope! Like a `Loading...` component when the request is loading, an `error` message when it's a bad request, and display the prices once the request succeeds.